package.js

```diff
@@ -1,35 +1,36 @@
 /* global Package */
 /* eslint-disable max-len,func-names,prefer-arrow-callback */
 Package.describe({
-  name: 'rlivingston:simple-schema-mixin',
-  version: '1.0.0',
-  summary: 'A ValidatedMethod mixin that eases use of aldeed SimpleSchema with mdg validated-method',
-  git: 'https://github.com/rhettlivingston/meteor-simple-schema-mixin.git',
-  documentation: 'README.md',
+    name: 'rlivingston:simple-schema-mixin',
+    version: '1.0.0',
+    summary: 'A ValidatedMethod mixin that eases use of aldeed SimpleSchema with mdg validated-method',
+    git: 'https://github.com/rhettlivingston/meteor-simple-schema-mixin.git',
+    documentation: 'README.md',
 });
 
-Package.onUse(function (api) {
-  api.versionsFrom('1.2');
```

Comment:
Will these changes still work with 1.2? I didn't think it had the necessary NPM support.